### ISSUE TITLE
Switching navigating panes with keyboard does not indicate which pane is active

### ISSUE THREAD

**Issue by erikjv** [owncloud/client#11791]
When using the narrator, and using the keyboard to navigate: when pressing "space" to switch from e.g. settings to an account, the narrator will only say "space", and not say that an account pane is now active.

**Comment by erikjv** [comment]
Selection should work different: when the tab "focus" is changed, the new tab should be foregrounded immediately

**Comment by TheOneRing** [comment]
The current bar is a mix of a tab and button bar, the current behavior is suboptimal, but any solution would require a conceptual redesign of the bar.
